tutorials/picca_export_stacked_correlation.py

```diff
@@ -75,20 +76,20 @@
 
     ### same header
     for i in range(nbData):
-        for k in ['NSIDE','HLPXSCHM','NP','NT','RTMAX','RPMIN','RPMAX']:
+        for k in ['NSIDE','HLPXSCHM','NPB','NTB','RTMAX','RPMIN','RPMAX']:
             assert data[i][k]==data[0][k]
```

Comment:
@andreufont, I do not think this is correct, since in the fits files the keys are "NP" and "NT".